src/logic/Preferences.swift

```diff
@@ -242,12 +242,12 @@ class Preferences {
     @available(OSX, deprecated: 10.11)
     private static func migrateLoginItem() {
         do {
-            let loginItems = LSSharedFileListCreate(nil, kLSSharedFileListSessionLoginItems.takeRetainedValue(), nil).takeRetainedValue()
-            let loginItemsSnapshot = LSSharedFileListCopySnapshot(loginItems, nil).takeRetainedValue() as! [LSSharedFileListItem]
+            let loginItems = LSSharedFileListCreate(nil, kLSSharedFileListSessionLoginItems.takeRetainedValue(), nil)!.takeRetainedValue()
+            let loginItemsSnapshot = LSSharedFileListCopySnapshot(loginItems, nil)!.takeRetainedValue() as! [LSSharedFileListItem]
             let itemName = Bundle.main.bundleURL.lastPathComponent as CFString
             let itemUrl = URL(fileURLWithPath: Bundle.main.bundlePath) as CFURL
             loginItemsSnapshot.forEach {
-                if (LSSharedFileListItemCopyDisplayName($0)?.takeRetainedValue() == itemName) ||
+                if (LSSharedFileListItemCopyDisplayName($0).takeRetainedValue() == itemName) ||
```

Comment:
I had to do some unwrapping here to get AltTab to compile. I'm not sure if my lint rules are different or if Xcode 14 has new nullability indicators.

Reply:
I think this is neither. I think it's the following: the SDK for more recent versions of macOS has updated these public APIs signature. And Apple has this terrible binding of certain macOS versions > certain XCode versions > certain SDK versions. So you're on Ventura, so you have to use a more recent XCode which has to use a more recent SDK, so you get the compiling errors.

Meanwhile i'm on Catalina still, and so is the CI machine. So both build fine.